nni/retiarii/oneshot/pytorch/darts.py

```diff
@@ -19,7 +20,7 @@ class DartsLayerChoice(nn.Module):
     def __init__(self, layer_choice):
         super(DartsLayerChoice, self).__init__()
         self.name = layer_choice.key
```

Comment:
should be layer_choice.label, otherwise there would be warning. DartsInputChoice should also be modified.